src/amd/compiler/aco_opcodes.py

```diff
@@ -599,12 +617,12 @@ def opcode(name, opcode_gfx9 = -1, format = Format.PSEUDO, input_mod = False, ou
    (0x1f, 0x1f, 0x16, 0x16, 0x1f, "v_mac_f32", True),
    (0x20, 0x20, 0x17, 0x17, 0x20, "v_madmk_f32", False),
    (0x21, 0x21, 0x18, 0x18, 0x21, "v_madak_f32", False),
-   (0x25, 0x25, 0x19, 0x19, 0x25, "v_add_co_u32", False),
-   (0x26, 0x26, 0x1a, 0x1a, 0x26, "v_sub_co_u32", False),
-   (0x27, 0x27, 0x1b, 0x1b, 0x27, "v_subrev_co_u32", False),
-   (0x28, 0x28, 0x1c, 0x1c, 0x28, "v_addc_co_u32", False),
-   (0x29, 0x29, 0x1d, 0x1d, 0x29, "v_subb_co_u32", False),
-   (0x2a, 0x2a, 0x1e, 0x1e, 0x2a, "v_subbrev_co_u32", False),
+   (0x25, 0x25, 0x19, 0x19,   -1, "v_add_co_u32", False),
+   (0x26, 0x26, 0x1a, 0x1a,   -1, "v_sub_co_u32", False),
+   (0x27, 0x27, 0x1b, 0x1b,   -1, "v_subrev_co_u32", False),
+   (0x28, 0x28, 0x1c, 0x1c,   -1, "v_addc_co_u32", False),
+   (0x29, 0x29, 0x1d, 0x1d,   -1, "v_subb_co_u32", False),
+   (0x2a, 0x2a, 0x1e, 0x1e,   -1, "v_subbrev_co_u32", False),
```

Comment:
V_SUBREV_CO_CI_U32

Reply:
Actually RDNA has V_SUBBREV_CO_U32 but it's VOP3B only, hence not yet included here.

Reply:
But sure I can add V_SUBREV_CO_CI_U32 here, same as its GFX7 opcode `0x2a`.